libbeat/docs/monitoring/monitoring-internal-collection.asciidoc

```diff
@@ -42,9 +43,11 @@ the same {es} cluster, specify the following minimal configuration:
 monitoring:
   enabled: true
   elasticsearch:
+    api_key:  id:api_key <1>
     username: {beat_monitoring_user}
     password: somepassword
 --------------------
+<1> Specify one of `api_key` or `username`/`password`.
```

Comment:
Related to my other comment.... This callout is ambiguous because you use `api_key` in two places (the setting name and example). I think you should use `id:key` here to avoid the ambiguity. We don't want users who skim read to set  `api_key: id:joe/password` (I've seen things).

Reply:
Hm, I would hope that seeing the configuration example above the callout (`api_key: TiNAGG4BaaMdaH1tRfuU:KnR6yE41RrSowb0kQ0HWoA`) would make it obvious that this does not contain either `id` or a username.

Reply:
I would hope that, too...but I've also seen people do lots of really weird things. Any ambiguity becomes a potential stumbling block, especially for people who don't read carefully.

I think your previous comment is spot-on, though. The real problem here is inherent in the names used in the API. So let's leave this as-is and see how it goes.